R/summarize_var_notaro.R

```diff
@@ -47,7 +47,7 @@ summarize_var_notaro <- function(nc.file, var.name){
       mutate(doy=as.numeric(DateTime-base.date)/86400+1) %>% 
       select(-DateTime, -tz, -base.date) %>% select(doy, everything()) %>% group_by(doy) %>%  
       summarize_each(c('mean','sd')) %>% 
-      setNames(c('doy',rename_depths(names(.)[-1L]))) %>% gather(key = doy) %>% 
+      setNames(c('doy',rename_depths(names(.)[-1L]))) %>% ##gather(key = doy) %>% 
```

Comment:
@jread-usgs is `gather` required here? Trying to deal with a weird issue on linux where it consistently fails here.